Dockerfile

```diff
@@ -18,8 +18,7 @@ RUN bundle install
 
 ADD . $APP_HOME
 
-ARG COMPILE_ASSETS=false
-RUN if [ "$COMPILE_ASSETS" = "true" ] ; then bundle exec rails assets:precompile ; fi
+RUN GOVUK_WEBSITE_ROOT=http://www.dev.gov.uk GOVUK_APP_DOMAIN=www.gov.uk RAILS_ENV=production bundle exec rails assets:precompile
```

Comment:
I guess for consistency it'd be good if the GOVUK_WEBSITE_ROOT matched GOVUK_APP_DOMAIN even though we don't expect them to be used.